packages/terra-search-field/src/SearchField.jsx

```diff
@@ -6,7 +6,7 @@ import ThemeContext from 'terra-theme-context';
 import Button from 'terra-button';
 import * as KeyCode from 'keycode-js';
 import IconSearch from 'terra-icon/lib/icon/IconSearch';
-import Input from 'terra-form-input';
+// import Input from 'terra-form-input';
```

Comment:
You forgot to remove this comment.

Reply:
Removed here https://github.com/cerner/terra-core/pull/3078/commits/e87bc5c5b9760c13bda762afd10b3f163229b3f5.